### PR TITLE
Endgame page

### PR DESCRIPTION
**Pull request by algibson1**
✨ What I did:
Added the EndGame page, including styling. Scoreboard currently has hardcoded placeholder data, which will be updated later to be dynamic when the GameBoard is updated to save scores in localStorage.

Did this break anything?

- [X] No
- [ ] Yes

Type of change

- [X] New feature (non-breaking change which adds functionality)
- [X]  Styling 
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [ ]  If this code needs to be tested, all tests are passing
- [X]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [ ]  The code is DRY. If it's not, I tried my best
- [ ]  I reviewed my code before pushing

✨ What's next?
Needs to be dynamic according to user's actual score. Testing also needs to be added.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -3,13 +3,15 @@ import './App.css';
...
 import {GameBoard} from 'Components/GameBoard/GameBoard';
+import EndGame from 'Components/EndGame/EndGame';
 
 function App() {
   return (
     <main className="App">
       <Routes>
         <Route path="/" element={<Welcome />}/>
         <Route path="/game/:theme" element={<GameBoard/>}/>
+        <Route path='/game_over' element={<EndGame/>}/>
```
[the remark this answers is not in the document]
Comment (ben-rosner-williamsburg) [reply src/App.tsx:14]: Looks like it's a no. Can we fix?

https://stackoverflow.com/questions/70551890/route-url-which-is-recommended-to-use-underscore-or-dash

**Review by ben-rosner-williamsburg — CHANGES_REQUESTED** [review]
Can we remove the underscore from the route path on Line 14 of App.tsx? Thanks!

File: src/Components/Welcome/Welcome.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ const Welcome = () => {
       </div>
       <div className='link-container'>
         <Link className="play-game-link" to={{pathname: `/game/${(JSON.parse(localStorage.getItem("selectedTheme") || '{}')).value || "random"}`}}>Play Game!</Link>
```
Comment (ben-rosner-williamsburg) [inline src/Components/Welcome/Welcome.tsx:19]: This Link tag should be self-closing. Meaning if we want the "Play Game!" text we'd have to wrap it in something else like a <p> tag.

**Review by ben-rosner-williamsburg — CHANGES_REQUESTED** [review]
Can we remove the underscore from the route path on Line 14 of App.tsx? Thanks!

File: src/App.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (algibson1) [reply src/App.tsx:14]: That's really interesting! The way that Ruby/Rails works, all the URLs I've ever written had underscores when there were multiple words. Now I'm curious to dive into some Rails forums to see if there are any debates about this.

File: src/Components/Welcome/Welcome.tsx (hunk 2 above)
Comment (algibson1) [reply src/Components/Welcome/Welcome.tsx:19]: Does this apply to the Link in `EndGame.tsx`?

**Review by ben-rosner-williamsburg — APPROVED** [review]
Thank you! Looks good now.

**Pull request merged**